apps/frontend/src/app/shared/feedback/error/error.component.html

```diff
@@ -13,7 +13,9 @@ <h1 class="font-bold">{{ title }}</h1>
         role="button"
         tabindex="0"
       >
+        @if (showAction){
```

Comment:
What does this serve? The button gets displayed neither way, just obce without text?